tasks/update.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Creates a folder /opt/mattermost/{{ mattermost_version }} with the desired version
+
+- set_fact:
+    mattermost_archive_name: mattermost-team-{{ mattermost_version }}
+  when: not mattermost_enterprise
+
+- set_fact:
+    mattermost_archive_name: mattermost-{{ mattermost_version }}
+  when: mattermost_enterprise
+
+- name: 'Download binary from https://releases.mattermost.com'
```

Comment:
I don't think this needs to be quoted.
Great idea to combine the tasks for both flavors!

Reply:
This line wasn't changed, but you're right.

Reply:
I know, I checked where the quotes were coming from